### DISCUSSION TITLE
iOS App does show accuracy only with 5m, although 10cm RTK accuracy in device

### DISCUSSION THREAD

**Discussion by FarmDoc** [opengisch/QField#4577] [Q&A]
I am using a Trimble DA2 Antenna and Catalyst 10 together with QField 2.8.3 in iOS 16.6.

Although I see RTK is locked and having 10cm Accuracy in the Trimble App (Mobile Manager) I do only have 5m horizontal accuracy in the Qfield App and therefore cannot properly use the accuracy options.
![image](https://github.com/opengisch/QField/assets/26462002/58d735a0-97d3-49ce-8b86-ca351fe3d54a)

This seems to be the standard behaviour for iOS because other Apps behave the same way or log it the same way (phyphox).

Is there really no solution to get around this limitation in iOS?

**Comment by w0pr** [comment]
I am having a similar issue on iOS tablet with the Leica ZenoConnect app and an external antennae: 
https://github.com/opengisch/QField/discussions/4756

**Comment by Archaeodave** [comment]
I have just got my Trimble DA2 and see the same thing in QField.  However the specified GNSS position changes with small movements of the antenna, and the speed displayed reacts if I move the antenna while keeping my iPhone stationary, so I wonder if QField is in fact picking up the corrected data but the accuracy data is not picked up from the Trimble Mobile Manager and simply reflects the iOS default for the iPhone.  The Trimble help suggests that if the iOS device just shows 5m then it's not picking up the corrected position, but if it's picking up the antenna movement while the phone is stationary it can't be solely using the inbuilt antenna.

**Comment by m-kuhn** [comment]
Trimble mentions at various places, that the position is correct, but the reported accuracy is not, e.g. https://community.trimble.com/discussion/using-catalyst-with-a-web-app-on-ios-platform

> Note that the position provided to iOS location services by the DA2 is accurate (see notes below), it's simply that iOS does not report that accuracy.

Instead, a WebSocket is often recommended, but without technical details about the protocol or connection. **If** this websocket talks NMEA over standard TCP or UDP, it should already work with QField.

**Reply by Archaeodave** [reply]
Thanks Matthias, that confirms what I thought.  I hadn't found the answer on Trimble Community Discussion so it's good to have that.
